docs/podman-tree.1.md

```diff
@@ -0,0 +1,33 @@
+% podman-tree "1"
```

Comment:
Does this need updating (throughout) for the `podman image tree` command name?

Reply:
Yes

Reply:
The file still needs to be renamed.

Reply:
Still outstanding.